scripts/import.py

```diff
@@ -143,7 +154,15 @@ def main(arguments: argparse.Namespace) -> None:
     else:
         backend = FakeTracBackend()
 
-    add(arguments.base, backend, arguments.year)
+    with contextlib.ExitStack() as stack:
+        if arguments.cache:
+            elements: Optional[Dict[str, int]] = stack.enter_context(
+                ElementsCache(arguments.cache),
+            )
+        else:
+            elements = None
```

Comment:
Should we just have this be a dict and avoid the differentiation?

Reply:
d7fa217